### PR TITLE
10 get/api/reviews,queries

### PR DESCRIPTION
**Pull request by Nayem59**
completed task 10 and ready for review

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -342,3 +342,95 @@ describe("GET /api/users", () => {
...
+      )
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review).toHaveProperty("category", category);
+        });
+        expect(reviews).toBeSortedBy(`${sort_by}`, {
+          descending: false,
+        });
```
Comment (Keb84) [inline __tests__/app.test.js:403]: This would be a good place to add one more 200 test that responds with an empty array if the category is valid but no articles in it 🤔

**Review by Keb84 — APPROVED** [review]
Once you've had a look at the comments for one more test feel free to merge ticket 10 🙂
